include/fmt/core.h

```diff
@@ -36,6 +36,9 @@
 #  define FMT_HAS_CPP_ATTRIBUTE(x) 0
 #endif
 
+#define FMT_HAS_CPP14_ATTRIBUTE(_attribute) (__cplusplus >= 201402L && FMT_HAS_CPP_ATTRIBUTE(_attribute))
```

Comment:
Why underscore in `_attribute`?

Reply:
I normally underscore macro parameters to help distinguish them from regular variable names.

That's not the normal fmt style though so I've removed them and run clang-format as requested.